File: app/src/main/java/com/thegreek/niko/logviewer_for_openHAB/TermsOfUse.java
package com.thegreek.niko.logviewer_for_openHAB;

import android.content.Intent;
import android.content.SharedPreferences;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.google.android.material.floatingactionbutton.FloatingActionButton;

import java.util.Objects;

public class TermsOfUse extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        SharedPreferences mySPR = this.getSharedPreferences("Safe", 0);
        MainActivity.changeOrientation(Objects.requireNonNull(this), mySPR.getInt("orientation", 0));
        setContentView(R.layout.activity_terms_of_use);

        TextView[] textViews = new TextView[7];

        for (int i = 1; i <= 7; i++) {
            textViews[i-1] = findViewById(getResources().getIdentifier("textView" + i, "id", getPackageName()));
            textViews[i-1].setText(getResources().getStringArray(R.array.terms_of_use)[i-1]);
        }

        FloatingActionButton fab = findViewById(R.id.fab);
        fab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent mailIntent = new Intent(Intent.ACTION_VIEW);
                Uri data = Uri.parse("mailto:?subject=" + getString(R.string.app_name) + " - Terms of Use"+ "&body=" + "" + "&to=" + "nikodiamond3@gmail.com");
                mailIntent.setData(data);
                startActivity(Intent.createChooser(mailIntent, getString(R.string.send_mail)));
            }
        });
    }
}

File: app/src/main/java/com/cyb3rko/logviewerforopenhab/TermsOfUse.kt
package com.cyb3rko.logviewerforopenhab

import android.content.Intent
import android.content.pm.ActivityInfo
import android.net.Uri
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.floatingactionbutton.FloatingActionButton

class TermsOfUse : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // load save file
        val mySPR = getSharedPreferences("Safe", 0)
        // restore set orientation
        requestedOrientation = mySPR.getInt("orientation", ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED)
        // set view
        setContentView(R.layout.activity_terms_of_use)

        // set text of textviews
        val textViews = arrayOfNulls<TextView>(7)
        for (i in 1..7) {
            textViews[i - 1] = findViewById(resources.getIdentifier("textView$i", "id", packageName))
            textViews[i - 1]?.text = (resources.getStringArray(R.array.terms_of_use)[i - 1])
        }

        // floating action button clicklistener
        val fab = findViewById<FloatingActionButton>(R.id.fab)
        fab.setOnClickListener { // start intent for writing an e mail
            val mailIntent = Intent(Intent.ACTION_VIEW)
            val data = Uri.parse(
                String.format(
                    getString(R.string.mail_info), getString(R.string.app_name),
                    resources.getStringArray(R.array.terms_of_use)[0]
                )
            )
            mailIntent.data = data
            startActivity(Intent.createChooser(mailIntent, getString(R.string.send_mail)))
        }
    }
}